server/hotrod/src/main/scala/org/infinispan/server/hotrod/OperationStatus.scala

```diff
@@ -30,6 +30,7 @@ object OperationStatus extends Enumeration {
    val OperationTimedOut = Value(0x86) // todo: test
    val NodeSuspected = Value(0x87)
    val IllegalLifecycleState = Value(0x88)
+   val Interrupted = Value(0x89)
```

Comment:
Unless you actually need a different behaviour from `IllegalLifecycleState`, I would prefer to see a single `ServerNotAvailable` operation status covering both. I know sometimes these exceptions can sometimes come from another node, but it should be the core's job to retry them and/or translate them to `SuspectException`.